rest/.jvm/src/main/scala/io/udash/rest/RestServlet.scala

```diff
@@ -70,9 +99,12 @@ class RestServlet(handleRequest: RawRest.HandleRequest, handleTimeout: FiniteDur
     restResponse.headers.entries.foreach {
       case (name, PlainValue(value)) => response.addHeader(name, value)
     }
-    restResponse.body.forNonEmpty { (content, mimeType) =>
-      response.setContentType(s"$mimeType;charset=utf-8")
-      response.getWriter.write(content)
+    restResponse.body.nonEmptyOpt.foreach { neBody =>
```

Comment:
Unrelated comment: HTTP responses should always set content length in servlets afair (at least for Jetty), regardless of how underlying implementation might handle it.